src/components/attribute/Attribute.tsx

```diff
@@ -0,0 +1,65 @@
+import { useAuth } from "@aha-app/aha-develop-react";
+import { LinkableRecord } from "@lib/linkableRecord";
+import { useClipboard } from "@lib/useClipboard";
+import React from "react";
+import { Branches } from "./Branches";
+import { EmptyState } from "./EmptyState";
+import { Menu } from "./Menu";
+import { PullRequests } from "./PullRequests";
+
+type AttributeProps = {
+  record: LinkableRecord;
+  fields: Github.IRecordExtensionFields;
+};
+
+export const Attribute: React.FC<AttributeProps> = ({ fields, record }) => {
+  const [onCopy, copied] = useClipboard();
+  const { error, authed } = useAuth(async () => {});
+  const authError = error && <div>{error}</div>;
+
+  const isLinked = [fields.branches, fields.pullRequests].some(
+    (ary) => ary && ary.length > 0
+  );
+
+  if (isLinked) {
+    return (
+      <div className="mt-1 ml-1">
+        <aha-flex
+          align-items="center"
+          justify-content="space-between"
+          gap="5px"
+        >
+          <Branches fields={fields} />
```

Comment:
This doesn't look right when there are no branches. Should we group the branches and PRs here instead?

![Screen Shot 2022-08-16 at 9 27 49 AM](https://user-images.githubusercontent.com/125077/184920345-b81f640f-852d-4ad4-98cd-3b90fcfb85cf.png)

```suggestion
          <div>
            <Branches fields={fields} />
            <PullRequests
              record={record}
              prs={fields.pullRequests}
            />
          </div>
```

Reply:
That's exactly why we added a heading in the Bitbucket PR (along with other UI enhancements to the list view).

<img width="649" alt="Screen Shot 2022-08-17 at 3 14 49 PM" src="https://user-images.githubusercontent.com/104908/185223795-64c722b5-165f-47c8-aa7c-ed54e108cb75.png">

Reply:
I've fixed the styling so that the menu can't drop to the left when there are no branches. I thought the heading looked a bit redundant, we shouldn't add it just to fix a styling issue, only if the information it's carrying is actually useful